spyder/widgets/tests/test_pathmanager.py

```diff
@@ -60,6 +62,57 @@ def test_check_uncheck_path(qtbot):
     assert pathmanager.not_active_pathlist == []
 
 
+@pytest.mark.skipif(os.name != 'nt',
```

Comment:
Please also skip this test if `pywin32` is not present because it's not part of our listed dependencies.

Reply:
For this I just have to write: `os.name != 'nt' or not pywin32`?

Reply:
You can use `programs.is_module_installed` for this.